### ISSUE TITLE
Bug: Accessibility not working

### ISSUE THREAD

**Issue by ben4d85** [pixijs/pixi-react#450]
### Current Behavior

I am trying to use the accessibility feature, but no HTML buttons are added to the DOM and no debug overlays are visible in the browser.

As far as I understand it, this should work by adding the `accessible` and `accessibleTitle` props to a `Sprite`. However, they appear to get ignored. Is there anything else needed? I am unable to find any example code and relevant docs for this?

My code:

```
 <Sprite
            ref={ref}
            image="https://s3-us-west-2.amazonaws.com/s.cdpn.io/693612/IaUrttj.png"
            anchor={0.5}
            scale={2}
            x={x}
            y={y + Math.sin(i) * 5}
            accessible={true}
            accessibleTitle="test"
            eventMode='dynamic'
            cursor='pointer'
            click={() => { console.log("Clicked");  }}
            {...props}
        />
```

### Expected Behavior

HTML buttons should be added to the DOM and visible in the browser's viewport and allow keyboard navigation via the Tab key.

### Steps to Reproduce

Add the following props to a `Sprite` component:

```
accessible={true}
accessibleTitle="test"
```

As far as I understand it, these two props should be all it takes?

### Environment

"pixi.js": "^7.2.4",
"pixi-viewport": "^5.0.1",
"@pixi/react": "^7.1.0",
"react": "18.2.0",
"react-dom": "18.2.0",
"next": "13.4.3",
"typescript": "5.0.4",

Chrome latest on MacOS.

**Comment by ben4d85** [comment]
**Update**

When using the Tab key multiple times, I can now see the debug overlays over each sprite in the browser and in the DOM. 

However, the key piece of information is that **the debug overlays only appear when the mouse cursor is inside the browser's viewport!**

They do not appear when the mouse cursor is outside of the browser's viewport, e.g. in Chrome Dev Tools. This means the overlays cannot be straightforwardly inspected in Chrome Dev Tools (it is however possible by using the keyboard instead of the mouse).

This is why I initially thought accessibility wasn't working. However, it does work.
